zerver/openapi/zulip.yaml

```diff
@@ -14922,9 +14931,13 @@ paths:
                             description: |
                               Unread count badge (appears in desktop sidebar and browser tab)
 
-                              - 1 - All unreads
-                              - 2 - Direct messages and mentions
-                              - 3 - None
+                              - 1 - All unread messages
+                              - 2 - DMs and mentions
+                              - 3 - DMs, mentions, and followed topics
+                              - 4 - None
+
+                              **Changes**: In Zulip 8.0 (feature level 227), added
+                              `DMs, mentions, and followed topics` option.
```

Comment:
"option, and renumbered the options." would be good here.

Reply:
Done